saylua/modules/items/models/db.py

```diff
@@ -24,6 +26,17 @@ def url(self):
 
     def image_url(self):
         return '/static/img/items/' + self.canon_name + '.png'
+        if is_devserver():
+            subpath = ("img" + os.sep + "items" + os.sep + self.canon_name + ".png")
+            image_path = (os.path.join( # The joys of going up five levels
```

Comment:
Make a utils function for this or something haha. This is going to drive me crazy otherwise.